src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -43,12 +48,31 @@
     @Argument(fullName = "disableSequenceDictionaryValidation", shortName = "disableSequenceDictionaryValidation", doc = "If specified, do not check the sequence dictionaries from our inputs for compatibility. Use at your own risk!", optional = true)
     private boolean disableSequenceDictionaryValidation = false;
 
-    @Argument(fullName="createOutputBamIndex", shortName="createOutputBamIndex", doc = "If true, create a BAM/CRAM index when writing a coordinate-sorted BAM/CRAM file", optional=true)
+    @Argument(fullName=StandardArgumentDefinitions.CREATE_OUTPUT_BAM_INDEX_LONG_NAME,
+            shortName=StandardArgumentDefinitions.CREATE_OUTPUT_BAM_INDEX_SHORT_NAME,
+            doc = "If true, create a BAM/CRAM index when writing a coordinate-sorted BAM/CRAM file.", optional=true)
     public boolean createOutputBamIndex = true;
```

Comment:
I disagree in this case, actually (though I agree with the general principle of not having boolean args that default to true, in general). If @cmnbroad inverted this arg, it would be inconsistent with `createOutputBamMD5`, etc., below, which would be confusing. Recommend keeping all of these args as `create*` instead of a mix of `dontCreate*` and `create*`